src/about-measure-z.md

```diff
@@ -41,7 +41,7 @@ Measure Z funds are distributed in the following ways:
 
 ### How much money is spent
 
-_[placeholder for dataviz]_
+{% include "dataviz-strategy-spending.njk" %}
```

Comment:
Liquid didn't like this for some reason, so I removed it, took the data-viz code out of `dataviz-strategy-spending.njk`, and pasted it directly into the `about-measure-z.md` page. Now it's working and everything is building again.